amqp.c

```diff
@@ -172,6 +173,8 @@ static PHP_MINIT_FUNCTION(amqp) /* {{{ */
 	REGISTER_LONG_CONSTANT("AMQP_OS_SOCKET_TIMEOUT_ERRNO",	AMQP_OS_SOCKET_TIMEOUT_ERRNO,	CONST_CS | CONST_PERSISTENT);
 	REGISTER_LONG_CONSTANT("PHP_AMQP_MAX_CHANNELS",			PHP_AMQP_MAX_CHANNELS,			CONST_CS | CONST_PERSISTENT);
 
+    REGISTER_LONG_CONSTANT("AMQP_SASL_METHOD_PLAIN",		AMQP_SASL_METHOD_PLAIN,		CONST_CS | CONST_PERSISTENT);
```

Comment:
formatting: tabs vs spaces, as this project uses tabs, for consistency purpose let's keep using tabs too.

BTW, I'm having this issue all the time too =).